Python/Automate_Commiting_using_Git/automate_commiting_using_git.py

```diff
@@ -1,7 +1,37 @@
 import subprocess
 
-subprocess.getoutput("git add .")
-message = input("Enter commit message")
-subprocess.getoutput("git commit -m " + message)
-branchname = input("Enter branch name")
-subprocess.getoutput("git push origin " + branchname)
+# Stage all changes
+result = subprocess.run(["git", "add", "."])
```

Comment:
`git add .` adds the current directory. Though this is ok for most use cases, it could be easier using gits pathspec: `:/`, really adding everything from the root of the git project, no matter what subdirectory someone is in. I think that would make it easier when dealing with large and deeply nested repositories (especially monorepos).

Reply:
Implemented.